x-pack/plugins/siem/server/graphql/overview/schema.test.ts

```diff
@@ -117,6 +120,9 @@ describe('SIEM Overview GQL Schema', () => {
       ecsSchema,
       overviewSchema,
       dateSchema,
+      toNumberSchema,
+      toDateSchema,
+      toBooleanSchema,
```

Comment:
Optional: Would it make sense to combine all our `ScalarSchema`'s into a single export  so when/if we add new ones we only have to add it in one place?

Ala what was done with the IpDetailsSchema [here](https://github.com/elastic/kibana/blob/7e60da4f87a1e2e8b67811aef7ae7a74d0d68dc6/x-pack/plugins/siem/server/graphql/index.ts#L33)